opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/MockTracingConfiguration.java

```diff
@@ -35,7 +38,9 @@
 
   @Bean
   public MockTracer mockTracer() {
-    return new MockTracer(new ThreadLocalActiveSpanSource(), Propagator.TEXT_MAP);
+    MockTracer mockTracer = new MockTracer(new ThreadLocalActiveSpanSource(), Propagator.TEXT_MAP);
+    setGlobal(mockTracer);
```

Comment:
This is not recommended. App should provide tracer bean or register global tracer. Just set it to Noop like in https://github.com/opentracing-contrib/java-spring-cloud/pull/64

Reply:
done